crates/matrix-sdk/src/docs/encryption.md

```diff
@@ -207,18 +207,18 @@ the device ID.
    This will replace the access token from the previous login call, but won't
    create a new device.
 
-**Note** that the default store supports only a single device, logging in
+**Note** that the default store supports only a single device. Logging in
 with a different device ID (either `None` or a device ID of another client)
 is **not** supported using the default store.
 
 ## Common pitfalls
 
-| Failure | Cause | Fix |
-| ------------------- | ----- | ----------- |
-| No messages get encrypted nor decrypted | The `e2e-encryption` feature is disabled | [Enable the feature in your `Cargo.toml` file] |
-| Messages that were decryptable aren't after a restart | Storage isn't setup to be persistent | Ensure you've activated the persistent storage backend feature, e.g. `sqlite` |
-| Messages are encrypted but can't be decrypted | The access token that the client is using is tied to another device | Clear storage to create a new device, read the [Restoring a Client] section |
-| Messages don't get encrypted but get decrypted | The `m.room.encryption` event is missing | Make sure encryption is [enabled] for the room and the event isn't [filtered] out, otherwise it might be a deserialization bug |
+| Failure                                               | Cause                                                               | Fix                                                                                                                            |
```

Comment:
What happened here? :scream:

Reply:
Argh, sorry - I've disabled my editor's format-on-save action for markdown.

Fixed in fd8c1e8dc2